### ISSUE TITLE
Is this library still maintained?

### ISSUE THREAD

**Issue by hkstrikers** [BrandonCookeDev/smashgg.js#83]
Looking for a smash.gg node js library, is this still actively maintained?

**Comment by rmaur012** [comment]
I think this one in particular is not being maintained anymore but I have forked this repo and worked on it to create the startgg npm library. You can find the repo on my account to take a look. Still needs work but it's available.

**Comment by hkstrikers** [comment]
@rmaur012 is your fork planned to be open sourced as well?

I'm part of the Mario Strikers community, and we currently have some integration built into a bot for Discord from start gg to help coordinate tournaments and store results into our backend. But would be interested in using a more fleshed out API than the raw Graph QL calls we're using now.

**Comment by rmaur012** [comment]
@hkstrikers yes it is to be open source. I think it's public for anyone to see and make PRs for.

I've made a few tweaks over time so that the tests passed so ideally they should be at a good point now but if you want to help make changes I'm down
